Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -710,6 +710,11 @@ open class LottieAnimationView: LottieAnimationViewBase {
     animationLayer?.getOriginalValue(for: keypath, atFrame: atFrame)
   }
 
+  /// Gets all child keypaths.
+  public func getHierarchyKeypaths() -> [String] {
```

Comment:
What use case do you have in mind for this API?

Reply:
I write the layers to a file. Having the layer names they can then be modified.

Reply:
What do you do with that file? Is there anything this lets you do that isn't possible today with `logHierarchyKeypaths()`?

Reply:
It allows making dynamic changes. Using logHierarchyKeypaths() means this would have to be done manually.